news/2020-07-06-new-beatmap-spotlights-season-summer-2020.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "New Beatmap Spotlights Season: Summer 2020"
+date: 2020-07-06 07:00:00 +0000
+---
+
+For the first time since our initial announcement, the renewed Beatmap Spotlights boot off with its first Summer Season and a plethora of rewards for everybody joining the brand-new competitive Seasonal League.
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+Everybody in our new team of curators worked hard to bring forth the most excellent maps osu! has to offer and to present you a new competitive league to enjoy their selections. While the first ideas have been kept vague in our first announcement, here are some details on how this new system works:
+
+- **Every weekend in this season we will be hosting a weekly timeshift lobby using a playlist created from the pool of curated beatmaps using osu!lazer.** Therefore, you have to join the timeshift lobbies on osu!lazer in order to participate in the competitive league. If you have never heard of timeshift multiplayer lobbies before, [you can learn more here](https://blog.ppy.sh/enjoying-osu-with-others/), or just [get osu!lazer yourself and try it out now](https://github.com/ppy/osu/releases).
+- Every Monday, the week's playlist will be unveiled on the [Beatmap Spotlights wiki page](/wiki/Beatmap_Spotlights). This means that **beatmaps being played in the Spotlights league lobbies will change every week**. For future seasons, we plan to unveil these maps in a livestream as a beatmap showcase.
+- The results of the weekly timeshift lobby will be collated into a big **Seasonal Ranking**. Depending your position on the Seasonal Ranking, you can earn yourself a **unique and permanent profile badge**. The profile badges will already be awarded after the 2nd week of this season and will be updated every week, reflecting your current position in the Seasonal Ranking.
+- There will be **four playlists in total, each consisting of 5 beatmaps** and each being played three times during the season. This gives you plenty of opportunities to revisit your weekly scores as only your best total score on a playlist will be considered for your position on the Seasonal Ranking.
+- On top of that **the Top 10 people of each timeshift lobby will be awarded with 1 week of supporter tag**.
+- At the end of the season, we will provide a **new unlockable medal** for those who want to play through the Seasonal Beatmaps once the season has concluded and **the best mapper of this season will be elected**. More information about that will follow at a later point.
```

Comment:
```suggestion
- At the end of the season,  **the best mapper of this season will be elected**, and we will provide a **new unlockable medal** for those who want to play through the Seasonal Beatmaps once the season has concluded. More information about this will follow at a later point.
```
This sentence flows better imo. "This" is a little ambiguous tho